File: FCUBEAPI/FreightMasters/Models/BusinessSummRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class BusinessSummRptListModel
    {
        public List<BusinessSummRptModel> BusinessSummRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/businesssummrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Businesssummrptmodel } from "./businesssummrptmodel";

export class Businesssummrptlistmodel {
    businessSummRptList: Businesssummrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
